test/integration/globals/default-route.js

```diff
@@ -23,7 +23,21 @@ context('patient page', function() {
 
     cy
       .url()
-      .should('contain', 'worklist/owned-by');
+      .should('contain', 'one/worklist/owned-by');
+  });
+
+  specify('current clinician has no workspaces', function() {
```

Comment:
This behavior is happening, but I _think_ it's throwing an error doing so.  We might actually want to attempt to force this to be true rather than depend on the error.  (and also clean up datadog)

Reply:
So remove this test from this PR and create a separate shortcut story?

Reply:
this is ok to me I think for now

Reply:
Okay I'll leave this test as-is for now.